Reflection/Login.simba

```diff
@@ -0,0 +1,299 @@
+type
+  TRSWorldScroll = (SCROLLWHEEL, SCROLLBAR, SCROLLARROWS);
+
+(*
+LOBBY INTERFACE
+================
+
+  Lobby Interface functionality.
+*)
+
+{*
+R_IsLobbyOpen
+~~~~~~~~~~~~~~~~~~~~~~
+Checks to see if the Lobby is open.
+*}
+Function R_IsLobbyOpen: Boolean;
+begin
+  Result := RSWidget.IsValid(R_POST_LOGIN_CLICK_TO_PLAY_CONTAINER.Group, R_POST_LOGIN_CLICK_TO_PLAY_CONTAINER.Child);
+end;
+
+Function R_WaitIsLobbyOpen(timeout: UInt32): Boolean;
+begin
+  Result := WaitUntil(R_IsLobbyOpen, 50, timeout);
+end;
+
+{*
+R_EnterGame
+~~~~~~~~~~~~~~~~~~~~~~
+Checks if we are in the Lobby and clicks the 'Click here to play' button if true.
+*}
+Function R_EnterGame(): Boolean;
+var
+  Widget: RSWidget;
+begin
+  if (not R_IsLobbyOpen) or R_WidgetIsHidden(R_POST_LOGIN_CLICK_TO_PLAY_CONTAINER) then
+    Exit(False);
+
+  Widget := RSWidget.Get(R_POST_LOGIN_CLICK_TO_PLAY_CONTAINER.Group, R_POST_LOGIN_CLICK_TO_PLAY_CONTAINER.Child);
+  if Widget.ref <> nil then
+  begin
+    Mouse.Click(Widget.Bounds, MOUSE_LEFT);
+    Result := True;
+    Widget.Free;
+  end;
+end;
+
+(*
+LOGOUT INTERFACE
+================
+
+  Logout Tab functionality.
+*)
+
+{*
+R_Logout
+~~~~~~~~~~~~~~~~~~~~~~
+Logs the player out of the game.
+
+Example:
+    R_Logout;
+*}
+Function R_Logout: Boolean;
+var
+  Widget: RSWidget;
+begin
+  if (not R_IsGametabOpen(GAMETAB_LOGOUT)) then
+    R_OpenGametab(GAMETAB_LOGOUT);
```

Comment:
Same as before - I think we can get away with just R_OpenGametab(GAMETAB_LOGOUT), right?